openmc/universe.py

```diff
@@ -713,6 +716,57 @@ def create_xml_subelement(self, xml_element, memo=None):
         dagmc_element.set('filename', self.filename)
         xml_element.append(dagmc_element)
 
+    def bounding_region(self, bounded_type='box', boundary_type='vacuum'):
+        """Creates a either a spherical or box shaped bounding region around
+        the DAGMC geometry.
+        Parameters
+        ----------
+        bounded_type : str
+            The type of bounding surface(s) to use when constructing the region.
+            Options include a single spherical surface (sphere) or a rectangle
+            made from siz planes (box).
+        boundary_type : str
+            Boundary condition that defines the behavior for particles hitting
+            the surface. Defaults to vacuum boundary condition. Passed into the
+            surface construction.
+        Returns
+        -------
+        openmc.Region
+            Region instance
+        """
+
+        check_type('boundary type', boundary_type, str)
+        check_value('boundary type', boundary_type, _BOUNDARY_TYPES)
+
+        bounding_box = self.bounding_box
+
+        if bounded_type == 'sphere':
```

Comment:
Can we add type/value checks for `bounded_type`?

Reply:
good idea, I've just added type and value checks in commit https://github.com/openmc-dev/openmc/pull/2118/commits/839b862d5645be785fd1f79dc6c3ba3f19e39b9e